Modules/Remote/TubeTK.remote.cmake

```diff
@@ -11,11 +11,11 @@
 #--
 #-- ## Compliance Level 4 star (Very high-quality code, perhaps small community dependance)
 #--   - [ ] Meets all ITK code style standards
-#--   - [ ] No external requirements beyond those needed by ITK proper
-#--   - [ ] Builds and passes tests on all supported platforms within 1 month of each core tagged release
-#--            - [ ] Windows Shared Library Build with Visual Studio
-#--            - [ ] Mac with clang compiller
-#--            - [ ] Linux with gcc compiler
+#--   - [X] No external requirements beyond those needed by ITK proper
```

Comment:
I assume this means cuFFT is optional?

Reply:
Yes - but it is already an option for ITK (I removed ArrayFire since it wasn't already an option for ITK).

Reply:
Sounds good.